### PR TITLE
Implementing add Todo Task, Logic & Storage

### PR DESCRIPTION
**Pull request by alfredkohhh**

### CODE REVIEW

File: src/main/java/manageezpz/logic/parser/CliSyntax.java [hunk 1]
```diff
@@ -9,7 +9,11 @@
     public static final Prefix PREFIX_NAME = new Prefix("n/");
     public static final Prefix PREFIX_PHONE = new Prefix("p/");
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
+    public static final Prefix PREFIX_DESCRIPTION = new Prefix("desc/");
+    public static final Prefix PREFIX_DATETIME = new Prefix("by/");
+    public static final Prefix PREFIX_TIME = new Prefix("at/");
     public static final Prefix PREFIX_TODO = new Prefix("/todo");
     public static final Prefix PREFIX_EVENT = new Prefix("/event");
```
Comment (aaron-ljx) [inline src/main/java/manageezpz/logic/parser/CliSyntax.java:16]: just a nitpick maybe but why the sudden format change in the prefix? prev was "by\" for example but now it's "/todo".

File: src/main/java/manageezpz/logic/commands/AddTodoTaskCommand.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    /**
+     * Creates an AddEmployeeCommand to add the specified {@code Person}
+     */
+    public AddTodoTaskCommand(Task task) {
+        requireNonNull(task);
+        toAdd = task;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
```
Comment (Denniszedead) [inline src/main/java/manageezpz/logic/commands/AddTodoTaskCommand.java:39]: Does this method require comments

File: src/main/java/manageezpz/logic/parser/AddTodoTaskCommandParser.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    /**
+     * Parses the given {@code String} of arguments in the context of the AddEmployeeCommand
+     * and returns an AddEmployeeCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public AddTodoTaskCommand parse(String args) throws ParseException {
+        ArgumentMultimap argMultimapTodo =
+                ArgumentTokenizer.tokenize(args, PREFIX_TODO, PREFIX_DESCRIPTION);
+
+        if (!arePrefixesPresent(argMultimapTodo, PREFIX_TODO, PREFIX_DESCRIPTION)
```
Comment (Denniszedead) [inline src/main/java/manageezpz/logic/parser/AddTodoTaskCommandParser.java:24]: Try splitting the boolean expression into local constants instead. It'll definitely make it more readable 👍

File: src/main/java/manageezpz/model/util/SampleDataUtil.java [hunk 4]
```diff
@@ -22,11 +25,22 @@
         };
     }
 
+    public static Task[] getSampleTasks() {
+        return new Task[] {
+            new Todo(new Description("Read Book")),
+            new Todo(new Description("Return Book")),
+            new Todo(new Description("Read Genshin Guides")),
+            new Todo(new Description("Do Daily Commissions")),
```
Comment (Denniszedead) [inline src/main/java/manageezpz/model/util/SampleDataUtil.java:33]: Should they be declared as constant?

File: src/main/java/manageezpz/logic/parser/CliSyntax.java (hunk 1 above)
Comment (ChanWeiJie) [reply src/main/java/manageezpz/logic/parser/CliSyntax.java:16]: Hi aaron, there is some issues with the PR being reflected but not to worry. We would still stick with the format of "type/", "by/", "at/" etc
eg: addTask todo/ desc/read book
Sorry for any confusion!

**Review by ChanWeiJie — APPROVED** [review]
LGTM! Thanks Alfred!

**Pull request merged**